### DISCUSSION TITLE
Welcome to Rubicon ObjC!

### DISCUSSION THREAD

**Discussion by freakboy3742** [beeware/rubicon-objc#207] [General]
## 👋 Welcome!

Rubicon ObjC is part of the [BeeWare Project](https://beeware.org).

This discussion forum is a place where you can:
* Ask questions about how to use Rubicon ObjC to bridge between Python and the Objective C runtime
* Discuss suggestions for new features, or changes to existing features
* Share details about projects where you've used Rubicon ObjC
* ...and anything else related to Rubicon ObjC that needs to be discussed!

We want this to be a welcoming space for all, so this server (along with all BeeWare community spaces) is subject to the [BeeWare Community Code of Conduct](https://beeware.org/community/behavior/).

If you want to say "Hi!", comment below with an introduction of yourself and tell us about what you do with Rubicon ObjC.

**Comment by nsmith1024** [comment]
Can I use it to call Python from Objective-c code?

**Reply by freakboy3742** [reply]
No; Rubicon is for the other direction - calling Objective C from Python code.

Standard CPython contains everything you need to call the other way - the C embedding API of Python allows you to instantiate a Python interpreter and run Python code. You can see an example of this in the [Briefcase template](https://github.com/beeware/briefcase-macOS-Xcode-template/blob/dev/%7B%7B%20cookiecutter.formal_name%20%7D%7D/%7B%7B%20cookiecutter.formal_name%20%7D%7D/main.m) for a macOS app.
